airflow/providers/amazon/aws/sensors/sqs.py

```diff
@@ -50,13 +52,15 @@ def __init__(
         aws_conn_id: str = 'aws_default',
         max_messages: int = 5,
         wait_time_seconds: int = 1,
+        message_attribute_names: list = ['.*']
```

Comment:
Let's push it under the constructor
```suggestion
        message_attribute_names: Optional[List[str]] = None
```

And then within the constructor
```python
self.message_attribute_names = ['.*'] if message_attribute_names is None else message_attribute_names 
```

The reason being python has gotcha around [mutable default arguments](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)